### PR TITLE
Task1 Чужой код

### PR DESCRIPTION
**Pull request by syberpunkq**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class ApplicationController < ActionController::Base
+  require 'pry'
```
Comment (X1ting) [inline app/controllers/application_controller.rb]: are you need this?

File: app/models/card.rb [hunk 2]
```diff
@@ -44,6 +45,17 @@ def self.pending_cards_notification
...
+      card = user.current_block.cards.pending.first
+      card ||= user.current_block.cards.repeating.first
+    else
+      card = user.cards.pending.first
+      card ||= user.cards.repeating.first
+    end
+    card
+  end
+
   protected
```
Comment (X1ting) [inline app/models/card.rb]: why not private?

File: app/models/card.rb [hunk 3]
```diff
@@ -44,6 +45,17 @@ def self.pending_cards_notification
     end
   end
 
+  def self.next_card(user)
```
Comment (X1ting) [inline app/models/card.rb]: how you can reduce size of this method? w/o moving logic to other methods? 
f.e. reduce size of this method to 4 lines.

File: app/models/card.rb (hunk 2 above)
Comment (syberpunkq) [reply app/models/card.rb]: fixed

File: app/models/card.rb (hunk 3 above)
Comment (syberpunkq) [reply app/models/card.rb]: done

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (syberpunkq) [reply app/controllers/application_controller.rb]: deleted

**Pull request merged**